feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/post/PostItem.kt

```diff
@@ -336,35 +367,49 @@ private fun HeadlineAuthorInfo(
             verticalAlignment = Alignment.CenterVertically,
             horizontalArrangement = Arrangement.spacedBy(4.dp)
         ) {
-            authorNameContent()
+            authorIconAndNameContent()
 
             creationDateContent()
         }
     }
 }
 
+@Composable
+private fun HeadlineProfilePicture(
```

Comment:
This composable does not have a root layout. I think it should either have a layout (Box, Column, etc.) or specify explicitly in which context it should be called.